simpl/rest.py

```diff
@@ -23,6 +23,77 @@
 STANDARD_QUERY_PARAMS = ('offset', 'limit', 'sort', 'q', 'facets')
 
 
+def One(schema=str):
```

Comment:
Did you know about [voluptuous.Length](https://github.com/alecthomas/voluptuous/blob/master/voluptuous.py#L1502)? It might return a more useful error.